scripts/make_orog.py

```diff
@@ -0,0 +1,132 @@
+"""
+The run script for making the orography files for the experiment.
+"""
+
+import glob
+import logging
+import os
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+
+from uwtools.api.filter_topo import FilterTopo
+from uwtools.api.orog import Orog
+from uwtools.api.orog_gsl import OrogGSL
+from uwtools.api.shave import Shave
+from uwtools.api.config import get_yaml_config
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+parser = ArgumentParser(
+    description="Script that runs the make_grid task via uwtools API",
+)
+parser.add_argument(
+    "-c",
+    "--config-file",
+    metavar="PATH",
+    required=True,
+    help="Path to experiment config file.",
+    type=Path,
+)
+parser.add_argument(
+    "--key-path",
+    help="Dot-separated path of keys leading through the config to the tasks's YAML block",
+    metavar="KEY[.KEY...]",
+    required=True,
+)
+
+args = parser.parse_args()
+expt_config = get_yaml_config(args.config_file)
+make_orog_config = get_yaml_config(expt_config[args.key_path])
+task_rundir = Path(make_orog_config["rundir"])
+print(f"Will run make_orog in {task_rundir}")
+
+CRES = expt_config["workflow"]["CRES"]
+os.environ["CRES"] = CRES
+
+KEY_PATH = args.key_path.split(".")
+fix_lam_path = Path(expt_config["workflow"]["FIXlam"])
+
+# Run orog
+orog_driver = Orog(
+    config=args.config_file,
+    key_path=[args.key_path],
```

Comment:
I definitely made some bad assumptions here that because in this iteration it's only one level deep, it would only ever be a single string with no dots. I'm making some mods to hopefully fix that poor assumption after your previous comment about splitting while parsing the args.